docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+
+[!code-csharp[ImageTransforms](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ImageTransforms)]
+
+The `TensorFlowTransform` extracts specified outputs and scores a dataset using a pre-trained Tensorflow model. 
+
+[!code-csharp[ScoreTensorFlowModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ScoreTensorFlowModel)]
+
+### Choose a learning algorithm
+
+To add the learning algorithm, call the `mlContext.MulticlassClassification.Trainers.LogisticRegression` wrapper method.  The `LogisticRegression` is appended to the `pipeline` and accepts the featurized `Title` and `Description` (`Features`) and the `Label` input parameters to learn from the historic data. You also need to map the label to the value to return to its original readable state. Do both of those actions with the following code:
+
+[!code-csharp[AddTrainer](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#AddTrainer)]
+
+You also need to map the predictedlabel to the predictedlabelvalue:
+
+[!code-csharp[MapValueToKey2](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#MapValueToKey2)]
+
+You train the model, <xref:Microsoft.ML.Data.TransformerChain%601>, based on the dataset that has been loaded and transformed. Once the estimator has been defined, you train your model using the <xref:Microsoft.ML.Data.EstimatorChain%601.Fit%2A> method while providing the already loaded training data. This returns a model to use for predictions. `pipeline.Fit()` trains the pipeline and returns a `Transformer` based on the `DataView` passed in. This DataView includes the `softmax2_pre_activation` property (also known as image features) and is produced by the ApplyTensorFlowGraph function. The experiment is not executed until the `.Fit()` method runs. Add this with the following code:
+
+[!code-csharp[TrainModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#TrainModel)]
+
+Notice the use of the `Transform` method of the machine learning `model` (a transformer) to input the features and return predictions.
+
+[!code-csharp[TransformData](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#TransformData)]
+
+
+
+[!code-csharp[EnumerateModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#EnumerateModel)]
+
+
+
+
+[!code-csharp[DisplayInfo](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#DisplayInfo)]
+
+The <xref:Microsoft.ML.MulticlassClassificationCatalog.Evaluate%2A> method computes the quality metrics for the model using the specified dataset. It returns a <xref:Microsoft.ML.Data.MultiClassClassifierMetrics> object that contains the overall metrics computed by multiclass classification evaluators.
```

Comment:
First time in the tutorial we mention Multi-Classification, this is something that should be presented at the start of the tutorial. 

We are building a customized image classifier using ImageNet. ImageNet as an image classifier allows us to predict between a 1000 categories, since this allows us to classify > 2 categories this is an example of multi-class classifier etc.